mock/cpp_mock_scenarios/src/traffic_source/define_traffic_source_delay.cpp

```diff
@@ -75,19 +75,17 @@ class DefineTrafficSourceDelay : public cpp_mock_scenarios::CppScenarioNode
         stop(cpp_mock_scenarios::Result::FAILURE);  // LCOV_EXCL_LINE
       }
       for (const auto & name : names) {
-        const traffic_simulator::CanonicalizedEntityStatus entity_status =
-          api_.getEntityStatus(name);
+        if (const auto entity = api_.getEntity(name)) {
+          const bool is_vehicle =
+            entity->getEntityType().type == traffic_simulator_msgs::msg::EntityType::VEHICLE;
```

Comment:
The same process is implemented in many scenarios.
For ease of maintenance, please implement it in the [base class for C++ scenarios](https://github.com/tier4/scenario_simulator_v2/blob/master/mock/cpp_mock_scenarios/include/cpp_mock_ scenarios/cpp_scenario_node.hpp) file.

Reply:
Please correct many of the same variables for scenarios that we have not pointed out.

Reply:
@hakuturu583 
Certainly, [here is commit](https://github.com/tier4/scenario_simulator_v2/pull/1320/commits/8425b2545bfe4cbd0819941fe9df3b8f909ce7e1) introduces changes :bow: